docs/source/tutorials/multilabel_classification.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
Can we replace: "multilabel" with "multi-label" in all the text?

Reply:
and should generally say: "multi-label classification datasets", not just "multi-label datasets", except for where the context is clear.

Reply:
Fixed: https://github.com/cleanlab/cleanlab/pull/517/commits/7aa53405e8abc5f3f0aa2f2a3a96f90f21516db1